lisa/tools/ip.py

```diff
@@ -127,6 +133,20 @@ def _get_matched_dict(self, result: str) -> Dict[str, str]:
             "ip_addr": matched.group("ip_addr"),
         }
 
+    def is_device_up(self, nic_name: str) -> bool:
+        device_info = self.node.execute(
+            f"ip -br link show {nic_name}",
```

Comment:
put it into ip tool

Reply:
Not sure what you mean, pull the regex into the tool?
or use self.run?